src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -496,4 +508,17 @@ private float CalculateStorageScore(MicrobeSpecies species, BiomeConditions biom
 
         return Math.Clamp(cacheScore, 0, Constants.AUTO_EVO_MAX_BONUS_FROM_ENVIRONMENTAL_STORAGE);
     }
+
+    /// <summary>
+    ///   Calculates cos of angle between the organelle and vertical axis
+    /// </summary>
+    private float CalculateAngleMultiplier(Hex pos)
+    {
+        Vector3 organellePosition = Hex.AxialToCartesian(pos);
+        Vector3 downVector = new Vector3(0, 0, 1);
+        float angleCos = organellePosition.Normalized().Dot(downVector.Normalized());
```

Comment:
```suggestion
        float angleCos = organellePosition.Normalized().Dot(downVector);
```

One more tiny suggestion, a slight performance improvement from not normalizing something that always has to be already normalized.